### ISSUE TITLE
Insert Timestamp 

### ISSUE THREAD

**Issue by shtalinberg** [tomboy-notes/tomboy-ng#108]
Insert Timestamp with shortcut   - useful functionality in previous  linux tomboy
can you add to new version ?

**Comment by davidbannon** [comment]
Yes, should be easy. Ctrl-T ?
What would you want stamped, that is what would you like the stamp to look like ?
Davo

**Comment by shtalinberg** [comment]
Ctrl-D (Cmd-D for mac)  better for compatibility with old tomboy
 you can try in previous linux tomboy version - looked good

**Comment by davidbannon** [comment]
I don't have Tomboy installed anymore, without Tomboy, I don't need the massive Mono libraries ....
If you can give me a sample, or something like -
[21 Aug, 2019 - 10:11]
I'd rather avoid getting into an argument with other sabout the right way to printing dates, the ISO standard,  20190821 is correct but ugly. My American cousins like 082119, incorrect and ugly :-)
What do you like ?
  
Davo

**Comment by dajoker** [comment]
I'll represent the Americans, since they'll just give bad ideas otherwise.  How about this:

yyyy-MM-dd HH:MM:SS
e.g.
2019-12-29 21:59:03

It's easy for somebody to reformat, and in the future the formatting could pull locale settings, or let the user set the format themselves, and for now we'd have something that is great.

**Comment by davidbannon** [comment]
That, sir, is the ansi standard !  My preferred way but everyone I speak to hates it. Its good in the  most significant terms are always to the left and because the year is obviously at the start, we know its not one of the other ambiguous possibilities. And easily generated in code.

**Comment by dajoker** [comment]
Yes, I agree with everything you wrote, though most people I talk do don't hate it, they just have to look at it a second time to figure it out.  Without it, though, we're stuck with the backwards europeans (day before month before year, so literally backwards) and the messed-up Americans (month-day-year), and that's before we get into a squabble about two-digit vs. four-digit years, or twelve-hour vs. twenty-four-hour clocks, or spelled out month names (argh no!), or "time" before "date", or the level of precision in the timestamp.  ANSI gives a good default, and then as people chime in, or volunteer to help (or maybe from the start if you are up to it) a setting can be added for them to select common options, or specify their own.

In summary, as a default ANSI makes the most sense (to me) to complete the feature request, but by all means let people make it different (wrong) however they like if time permits, or if they want to code that feature themselves.  ;-)

**Comment by aguador** [comment]
The ANSI standard is the only one that makes sense. Are you going to waste code, possible providing a sort on the underlying ANSI format while "translating" the visual into the other formats based on interface language or something else?

**Comment by davidbannon** [comment]
Done in V0.26
